src/components/Header/AppHeaderUser.tsx

```diff
@@ -7,14 +7,21 @@ import connectWalletImg from "img/ic_wallet_24.svg";
 
 import "./Header.css";
 import { isHomeSite, getAccountUrl } from "lib/legacy";
-import { isDevelopment } from "lib/legacy";
 import cx from "classnames";
 import { Trans } from "@lingui/macro";
 import NetworkDropdown from "../NetworkDropdown/NetworkDropdown";
 import LanguagePopupHome from "../NetworkDropdown/LanguagePopupHome";
-import { ARBITRUM, ARBITRUM_TESTNET, AVALANCHE, getChainName } from "config/chains";
+import {
+  ARBITRUM,
+  ARBITRUM_TESTNET,
+  AVALANCHE,
+  AVALANCHE_FUJI_TESTNET,
```

Comment:
probably matter of taste, fuji is the name of avalanche's testnet, so i guess it's enough to call it "AVALANCHE_FUJI", what do you think?